README.md

```diff
@@ -1 +1,36 @@
-Live cell timelapse apopstosis analysis.
+# Live cell timelapse apoptosis analysis.
+
+The goal of this project is to develop a pipeline to analyze timelapse images of live cells undergoing apoptosis. The pipeline will be able to detect cells, track them over time, and classify them as either apoptotic or non-apoptotic.
+Each dataset contains timelapse images of HeLa cells in a 96-well plate. These cells are treated with varying concentrations of staurosporine, a drug that induces apoptosis.
```

Comment:
What is a dataset in this project? Consider defining (or linking to) how the datasets are composed (i.e. as directories, sets of files, etc).

Reply:
Made this more clear